app/bundles/LeadBundle/Entity/LeadListRepository.php

```diff
@@ -906,6 +912,208 @@ public function getListFilterExpr($filters, &$parameters, QueryBuilder $q, $not
                     }
                     $groupExpr->add(sprintf('%s (%s)', $operand, $subqb->getSQL()));
                     break;
+                case 'hit_url_date':
+                case 'lead_email_read_date':
+                    $operand = ($func == 'between' || in_array($func, ['eq', 'gt', 'lt', 'gte', 'lte'])) ? 'EXISTS' : 'NOT EXISTS';
```

Comment:
`($func == 'between' || in_array($func, ['eq', 'gt', 'lt', 'gte', 'lte']))`
is the same as
`(in_array($func, ['eq', 'gt', 'lt', 'gte', 'lte', 'between']))`
Isn't it?